waterlog/src/reducers/SegmentEventsReducer.js

```diff
@@ -0,0 +1,36 @@
+import {
+    FETCH_SEGMENT_EVENTS_BEGIN, FETCH_SEGMENT_EVENTS_SUCCESS, FETCH_SEGMENT_EVENTS_FAILURE
```

Comment:
More than 2 should be multi line.

Reply:
Agree with Eduard

import {
    FETCH_SEGMENT_EVENTS_BEGIN, 
    FETCH_SEGMENT_EVENTS_SUCCESS, 
    FETCH_SEGMENT_EVENTS_FAILURE
} from '../actions/Types';

Reply:
see your **SegmentEventsActions.js** file. 
keep your coding convention consistent

Reply:
Sure I fixed it.